clang/docs/tools/dump_ast_matchers.py

```diff
@@ -101,7 +101,7 @@ def extract_result_types(comment):
 
 
 def strip_doxygen(comment):
-    """Returns the given comment without \-escaped words."""
+    """Returns the given comment without \\-escaped words."""
```

Comment:
I see lots of changes to comment blocks in this patch. Can we use `'''` instead to fix those warnings?

Reply:
No, `'''\-'''` is still SyntaxWarning in Python 3.12. It is possible to not escape with raw literals, but raw literals are used mostly to indicate that string is actually _raw_ (I've never seen raw docblocks). Also there is some consensus between highlighting tools that raw literals use regexp highlighting (in vscode, dandavison/delta, sharkdp/bat), that's why I escaped some strings selectively (e. g. in runCmd I did not use raw literals).
<img alt="image" src="https://github.com/llvm/llvm-project/assets/108563/deb36005-b0e5-459d-b936-cf644243774f">